integration/query_array_test.go

```diff
@@ -543,7 +508,7 @@ func TestQueryArrayAll(t *testing.T) {
 		},
 		"EmptyNested": {
 			filter:      bson.D{{"v", bson.D{{"$all", bson.A{bson.A{}}}}}},
```

Comment:
Probably outside of the scope, when we query with nested array filter, what is our expectation? maybe @AlekSi

Reply:
Nice catch, removed that test.